mmdet/core/evaluation/text_evaluation.py

```diff
@@ -57,6 +57,14 @@ def polygon_from_points(points):
     return plg.Polygon(point_mat)
 
 
+def refactor_to_4_points(points):
+    """Return coordinates of 4 points of the each corner instead of 2 corner points"""
```

Comment:
According to you code it is not conversion from 2 points to 4 points, it is x,y,w,h to 4 points